prototypes/styletransfer/lambda.py

```diff
@@ -1,63 +1,68 @@
 import json, boto3
-# This file is to store a lambda handler which gets triggered by an Amazon API Gateway POST method.  
-# A lambda handler is a serverless python function executed each time the trigger is pulled. 
+
+# This file is to store a lambda handler which gets triggered by an Amazon API Gateway POST method.
+# A lambda handler is a serverless python function executed each time the trigger is pulled.
 # In this specific case, the function does the following:
 # 1) spins up a p2.xlarge EC2 instance with a custom deep learning AMI
 # 2) clones our transfer learning repo
 # 3) launches the server side websocket
 # 4) kills the EC2 after 10 minutes
-# 5) returns to the browser the instance_id of the spinned up EC2 instance  
+# 5) returns to the browser the instance_id of the spinned up EC2 instance
+
+REGION = "eu-west-1"  # region to launch instance.
+AMI = "ami-0a961c5be1838f98e"  # our custom AMI
+INSTANCE_TYPE = "p2.xlarge"  # instance type to launch.
 
-REGION = 'eu-west-1' # region to launch instance.
-AMI = "ami-0a961c5be1838f98e" # our custom AMI
-INSTANCE_TYPE = 'p2.xlarge' # instance type to launch.
+EC2 = boto3.client("ec2", region_name=REGION)
 
-EC2 = boto3.client('ec2', region_name=REGION)
 
 def format_response(message, status_code):
     return {
-        'statusCode': str(status_code),
-        'body': json.dumps(message),
-        'headers': {
-            'Content-Type': 'application/json',
-            'Access-Control-Allow-Origin': '*'
-            }
-        }
+        "statusCode": str(status_code),
+        "body": json.dumps(message),
+        "headers": {
+            "Content-Type": "application/json",
+            "Access-Control-Allow-Origin": "*",
+        },
+    }
+
 
 def lambda_handler(event, context):
 
-    body = json.loads(event['body'])
-  
+    body = json.loads(event["body"])
+
     init_script = f"""#!/bin/bash
     cd /home/ubuntu
     git clone https://github.com/gabrielelanaro/ml-prototypes.git
     source activate tensorflow_p36
     cd ml-prototypes
-    python -m prototypes.styletransfer.app --address=0.0.0.0 --port=8000
+    # We shut down the system in 10 minutes if the server doesn't return earlier than that
     shutdown -h +10
+    python -m prototypes.styletransfer.app --address=0.0.0.0 --port=8000
```

Comment:
@FraPochetti note that we need a shutdown before and after the call to the server

Reply:
Why?

Reply:
because otherwise we have:
```
python -m prototypes.styletransfera.app # This gets stuck and never returns
shutdown -h +10 # This line will never be executed, which means the machine will not shutdown in 10 minutes
```

Reply:
ok